mgmt/rest/v2/config.go

```diff
@@ -159,3 +205,66 @@ func getPluginType(t string) (core.PluginType, error) {
 	}
 	return ityp, nil
 }
+
+// deletePluginConfigItemHelper builds different forms of request data into the way method deletePluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func deletePluginConfigItemHelper(r *http.Request) []string {
+	buf, _ := ioutil.ReadAll(r.Body)
```

Comment:
it should never run into the error unless it's malfunction. Good question though. I'll handle it.